_assignments/important_notes.md

```diff
@@ -1,18 +1,14 @@
 1. Read the submission instructions carefully!
 If you are missing some files, but we can still grade, you will lose 20%.
 If we cannot grade your work due to missing files, you will receive no credit.
-1. Oculus demos need a TA to be present in order to install them.
```

Comment:
Do they no longer need a TA to install Oculus demos?

Reply:
That sort of thing should really just go on the MP1 notes, since it’s only relevant to that and MP3

Reply:
That's fair